src/components/TablePagination/index.scss

```diff
@@ -1,4 +1,4 @@
-#main-content .usa-pagination {
+.usa-pagination {
     justify-content: left;
 
     &__page-no {
```

Comment:
You have to be really careful about removing styles blanketed across a component.  This is a side affect on another table: 

![Screenshot 2024-08-07 at 10 01 00 AM](https://github.com/user-attachments/assets/ee2b9725-009f-4ee9-b828-6f90265cc350)